app/sdk/roles/models/AppRole.java

```diff
@@ -0,0 +1,42 @@
+package sdk.roles.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import sdk.annotations.Attribute;
+import sdk.annotations.PrimaryKey;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AppRole {
+    @PrimaryKey
+    @JsonProperty("roleName")
+    private String roleName;
+    @JsonProperty("roleID")
+    @Attribute(index = 0)
+    private int roleID;
```

Comment:
Is there any way for this to come back from the core null?

Reply:
@karikittie There shouldn't be.